projects/js-packages/publicize-components/src/components/form/index.tsx

```diff
@@ -49,6 +53,7 @@ export default function PublicizeForm() {
 	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
 
 	const { feature_flags } = getSocialScriptData();
+	const showSocialPostModal = feature_flags.useEditorPreview && isPublicizeEnabled;
```

Comment:
I think we not use this feature flag now. Otherwise, it means that we have removed resharing for the sites that don't have this feature yet for being on an older version of one of the two plugins which is not going to be the case as it has been a long time since we released Social Post UI.

Reply:
Good point done